### DISCUSSION TITLE
AudiOrbits on Lively

### DISCUSSION THREAD

**Discussion by Hialurono-Rugstis** [rocksdanister/lively#1202] [Q&A] [answered]
There is this amazing audio visualizer for Wallpaper Engine called [AudiOrbits](https://github.com/hexxone/audiorbits). Is it possible to make it work on Lively?

**Comment by rocksdanister** [comment]
I actually did do it in the past; due to this issue https://github.com/rocksdanister/lively/issues/15 the audio reaction is not great at that time.

**Reply by Hialurono-Rugstis** [reply]
Would you be able to share it?

**Reply by rocksdanister** [reply]
Sure, let me clean it up first..

**Comment by rocksdanister** [comment]
You can get it here (incomplete):
https://github.com/rocksdanister/audiorbits
(For now it also needs the project.json file in the root folder.)

Require Lively v2.0.3.2 minimum:
https://github.com/rocksdanister/lively/releases/tag/v2.0.3.2

**Reply by luftwafflezz** [reply]
How do you install this into Lively? I've tried to clone it and install it but it always errors out.

**Reply by rocksdanister** [reply]
LivelyInfo.json needs to be in the root folder of the zip.

**Reply by luftwafflezz** [reply]
Ok, it goes into Lively, but then this happens.
<img width="368" alt="Screenshot 2022-09-09 090444" src="https://user-images.githubusercontent.com/72886318/189356749-f4f2acbb-7154-4a01-b22d-04cfd43190cc.png">

**Reply by rocksdanister** [reply]
You are suppose to add the .zip (Add wallpaper -> Choose file) file with `LivelyInfo.json` and stuff in root directory, not individual file.

**Reply by luftwafflezz** [reply]
Oh, ok. I did that and now the preview gif plays, I select it and now the wallpaper is black and doesn't respond to audio, even when I play some music.

**Reply by rocksdanister** [reply]
I just pushed an update with a fix, should work now:
https://github.com/rocksdanister/audiorbits/commit/34e6226e0033898641e02a544deff0339d260bc7

Create new wallpaper zip or just edit the file.

**Reply by luftwafflezz** [reply]
Great! It works, thank you!

**Comment by rocksdanister** [comment, marked as answer]
I added some basic customisation support, can download from release:
https://github.com/rocksdanister/audiorbits/releases
